src/java.xml/share/classes/com/sun/org/apache/xpath/internal/NodeSetDTM.java

```diff
@@ -1050,7 +1050,7 @@ public int indexOf(int elem)
    * you call getCurrentPos() and the return is 0, the next
    * fetch will take place at index 1.
    *
-   * @return The the current position index.
+   * @return The current position index.
```

Comment:
Note that there is an inconsistent change here, albeit insignificant. In `NodeSet`, the capital 'The' is removed after the `@return`, while the lowercase 'the' is removed here.

Reply:
Thanks for pointing out the inconsistency.  I have fixed it.  (I noticed many violations of the official Javadoc style, but this is not the right pull request to fix them.)